### PR TITLE
update to libgsf-1.14.44 - add test suite

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: components/library/libgsf/Makefile [hunk 1]
```diff
@@ -8,7 +8,6 @@
 # source.  A copy of the CDDL is also available via the Internet at
 # http://www.illumos.org/license/CDDL.
 #
-
```
Comment (pyhalov) [inline components/library/libgsf/Makefile]: Avoid removing this line

Comment (ghost) [reply components/library/libgsf/Makefile]: Does it trigger something in the build system?

Comment (pyhalov) [reply components/library/libgsf/Makefile]: No, just standard header format from https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/templates/configure.mk

**Comment by ghost** [comment]
Updated file from web - Not sure about squash commits from there

**Pull request merged**